.github/workflows/test.yml

```diff
@@ -0,0 +1,79 @@
+name: "Test"
+on:
+    push:
+        branches:
+            - master
+        tags:
+            - "*"
+    pull_request:
+        types: [opened, reopened, synchronize]
+jobs:
+    unit:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                node: [8, 10, 11, 12]
```

Comment:
Does 13 also work by any chance? (Was reminded about this possibility by #3473....)

Reply:
yes i'll add it!

Reply:
looks like ran into the same error: https://github.com/ethereum/web3.js/pull/3468/checks?check_run_id=590277057#step:4:64

Reply:
I've seen the same problem in Ubuntu 18.04 with node 13. I think this is a ganache-cli problem.